discordbot.js

```diff
@@ -2778,7 +2778,7 @@ exports.createDiscordClient = (web3, contracts, getStores, runSidechainTransacti
                                   }
                               }).promise();
 
-                              const m = await message.author.send(`Play: https://webaverse.com/login?id=${id}&code=${code}&play=true`);
+                              const m = await message.author.send(`Play: https://app.webaverse.com/?id=${id}&code=${code}&play=true`);
```

Comment:
Wrong URL. It should be `https://app.webaverse.com/login`

Reply:
URL issue is resolved.